backend/src/main/java/com/bakdata/conquery/models/forms/managed/EntityDateQueryPlan.java

```diff
@@ -0,0 +1,83 @@
+package com.bakdata.conquery.models.forms.managed;
+
+import com.bakdata.conquery.apiv1.forms.FeatureGroup;
+import com.bakdata.conquery.apiv1.forms.Form;
+import com.bakdata.conquery.apiv1.forms.export_form.ExportForm;
+import com.bakdata.conquery.models.common.CDateSet;
+import com.bakdata.conquery.models.common.Range;
+import com.bakdata.conquery.models.common.daterange.CDateRange;
+import com.bakdata.conquery.models.forms.util.DateContext;
+import com.bakdata.conquery.models.query.QueryExecutionContext;
+import com.bakdata.conquery.models.query.entity.Entity;
+import com.bakdata.conquery.models.query.queryplan.ArrayConceptQueryPlan;
+import com.bakdata.conquery.models.query.queryplan.QueryPlan;
+import com.bakdata.conquery.models.query.queryplan.clone.CloneContext;
+import com.bakdata.conquery.models.query.results.ContainedEntityResult;
+import com.bakdata.conquery.models.query.results.EntityResult;
+import com.bakdata.conquery.models.query.results.MultilineContainedEntityResult;
+import lombok.RequiredArgsConstructor;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+public class EntityDateQueryPlan implements QueryPlan {
+
+
+    private final QueryPlan query;
+    private final ArrayConceptQueryPlan features;
+    private final List<ExportForm.ResolutionAndAlignment> resolutionsAndAlignments;
+    private final CDateRange dateRange;
+
+    @Override
+    public EntityResult execute(QueryExecutionContext ctx, Entity entity) {
+        EntityResult preResult = query.execute(ctx, entity);
+        if (preResult.isFailed() || !preResult.isContained()) {
+            return preResult;
+        }
+        final List<Object[]> resultLines = new ArrayList<>();
+        for( Object[] line : preResult.asContained().listResultLines()) {
+
+            CDateSet entityDate = CDateSet.parse((String) line[0]);

```

Comment:
Was passiert, wenn das eine 2ndId ist?

Reply:
Evtl musst du  den Index in der Query tracken, oder die 2nd id tauschen mit dem SpecialDateUnion